### ISSUE TITLE
[IDEA] Enable administrator font configuration

### ISSUE THREAD

**Issue by thisispaco** [nextgenhealthcare/connect#4780]
Expose UIConstants font parameters to the administrator configuration to be able to change font size and type.

// fonts
public static final String MONOSPACED_FONT_NAME = SystemUtils.IS_OS_MAC ? "Courier" : "Monospaced";
public static final Font TEXTFIELD_PLAIN_FONT = new Font("Tahoma", Font.PLAIN, 11);
public static final Font TEXTFIELD_BOLD_FONT = new Font("Tahoma", Font.BOLD, 11);
public static final Font BANNER_FONT = new Font("Arial", Font.BOLD, 36);
public static final Font DIALOG_FONT = new Font("Dialog", Font.PLAIN, 12);
public static final Font MONOSPACED_FONT = new Font(MONOSPACED_FONT_NAME, Font.PLAIN, 12);

**Comment by jonbartels** [comment]
I cannot find links quickly but this may be relevant to some long standing historic issues for:
* Support for non-western languages
* Issues with MS fonts (like Tahoma) on Linux systems (easy workaround to install a font pack though)

Letting the user change fonts may help mitigate those other issues.

**Comment by tonygermano** [comment]
> * Issues with MS fonts (like Tahoma) on Linux systems (easy workaround to install a font pack though)

It's actually not an easy workaround. Tahoma is not a free font and previous workarounds no longer work.

The problem with the Linux replacement font for Tahoma - or changing fonts in general - is that many labels and data grid columns are sized to fit the hard-coded font and are not adjustable. This leads to things being truncated and difficult to read.

**Comment by tonygermano** [comment]
Related to https://github.com/nextgenhealthcare/connect/issues/4527, https://github.com/nextgenhealthcare/connect/issues/4019, https://github.com/nextgenhealthcare/connect/issues/2129

Possible duplicate of https://github.com/nextgenhealthcare/connect/issues/3251 and https://github.com/nextgenhealthcare/connect/issues/2582, the second of which I already requested to be reopened

**Comment by flugruger** [comment]
> Expose UIConstants font parameters to the administrator configuration to be able to change font size and type.
> 
> // fonts public static final String MONOSPACED_FONT_NAME = SystemUtils.IS_OS_MAC ? "Courier" : "Monospaced"; public static final Font TEXTFIELD_PLAIN_FONT = new Font("Tahoma", Font.PLAIN, 11); public static final Font TEXTFIELD_BOLD_FONT = new Font("Tahoma", Font.BOLD, 11); public static final Font BANNER_FONT = new Font("Arial", Font.BOLD, 36); public static final Font DIALOG_FONT = new Font("Dialog", Font.PLAIN, 12); public static final Font MONOSPACED_FONT = new Font(MONOSPACED_FONT_NAME, Font.PLAIN, 12);

I agree that the Administrator tool would be better if it supported font changes. 

In the meantime, are these UIConstants somewhere I can manually edit?
